--- moc/decoder_plugins/sidplay2/sidplay2.cc.fix	2009-08-24 17:24:48 +0400
+++ moc/decoder_plugins/sidplay2/sidplay2.cc	2009-08-24 17:25:18 +0400
@@ -259,9 +259,9 @@ extern "C" void *sidplay2_open(const cha
     st->selectSong(s);
 
     if(!(*st))
     {
-      decoder_error(&s2d->error, ERROR_FATAL, 0, "Error determining length...", file);
+      decoder_error(&s2d->error, ERROR_FATAL, 0, "Error determining length of %s...", file);
       delete st;
       return s2d;
     }
 
