diff -urN moc-2.5.0-alpha4.orig/decoder_plugins/ffmpeg/ffmpeg.c moc-2.5.0-alpha4/decoder_plugins/ffmpeg/ffmpeg.c
--- moc-2.5.0-alpha4.orig/decoder_plugins/ffmpeg/ffmpeg.c	2010-04-26 17:38:03.496413812 +0400
+++ moc-2.5.0-alpha4/decoder_plugins/ffmpeg/ffmpeg.c	2010-04-26 17:38:59.461414964 +0400
@@ -25,7 +25,7 @@
 #if HAVE_LIBAVFORMAT_AVFORMAT_H
 #include <libavformat/avformat.h>
 #else
-#include <ffmpeg/avformat.h>
+#include <ffmpeg/libavformat/avformat.h>
 #endif
 
 /* FFmpeg also likes common names, without that, our common.h and log.h would
